includes/gateways/offline-donations.php

```diff
@@ -360,28 +360,25 @@ function give_get_default_offline_donation_content() {
  *
  * Gets the default offline donation email content
  *
+ * @unreleased Remove unnecessary sprintf
+ *
  * @return string
  */
 function give_get_default_offline_donation_email_content() {
 	$default_text  = '<p>' . __( 'Hi {name},', 'give' ) . '</p>';
 	$default_text .= '<p>' . __( 'Thank you for letting us know that you\'re mailing a check! Your generosity is greatly appreciated. Here are those steps again:', 'give' ) . '</p>';
 	$default_text .= '<ol>';
 	$default_text .= '<li>';
-	$default_text .= sprintf(
-		/* translators: %s: site name */
-		__( 'Write a check payable to "{sitename}"', 'give' )
-	);
+	$default_text .= esc_html__( 'Write a check payable to "{sitename}"', 'give' );
```

Comment:
This one might still need to be a `sprintf` with `{sitename}` passed.

Although, this does match the current translation and changing it would break existing translations... So, I'll leave this as a note in case it becomes an issue.

Reply:
@kjohnson we want to print it as an email tag in the email message setting, we do not was anyone to translate it directly.

Reply:
Yeah, which is why I was thinking that it should be passed in to replace a placeholder, ie `%s`, since it isn't translatable.

```php
sprintf(
    esc_html( 'Write a check payable to "%s"', 'give' ),
    '{sitename}'
)
```

I think this would be the correct way to use a merge tag inside of a translatable string.

However, since this is an existing translatable string, changing the structure would break existing translations, so I don't think that we should change it right now.

Reply:
@kbjohnson90 agree with you